docs/content/en/docs/Coding Milestones/PoC2023q1/outline.md

```diff
@@ -22,9 +22,9 @@ progress.
 This is a quick demo of a fragment of what we think is needed for edge
 multi-cluster.  It is intended to demonstrate the following points.
 
-- Separation of inventory and workload management.
+- Separation of infrastructure and workload management.
```

Comment:
Do we need an "inventory" for the workloads as well? (i.e., separate the workload management into modules) or we define that as part of the overall "workload management"

Reply:
The API objects in the workload management workspaces are the inventory of workload.